tests/test_mlip_calculators.py

```diff
@@ -2,63 +2,137 @@
 
 from pathlib import Path
 
+from chgnet.model.model import CHGNet
+from matgl import load_model
 import pytest
 
 from janus_core.helpers.mlip_calculators import choose_calculator
 
-MODEL_PATH = Path(__file__).parent / "models" / "mace_mp_small.model"
-
-test_mlips_data = [
-    ("mace", "cpu", {"model": MODEL_PATH}),
-    ("mace", "cpu", {"model_paths": MODEL_PATH}),
-    ("mace_off", "cpu", {}),
-    ("mace_mp", "cpu", {}),
-    ("mace_mp", "cpu", {"model": MODEL_PATH}),
-    ("mace_off", "cpu", {"model": "small"}),
-    ("m3gnet", "cpu", {}),
-    ("chgnet", "cpu", {}),
-]
-
-
-@pytest.mark.parametrize("architecture, device, kwargs", test_mlips_data)
+MACE_MP_PATH = Path(__file__).parent / "models" / "mace_mp_small.model"
+
+MACE_OFF_PATH = Path(__file__).parent / "models" / "MACE-OFF23_small.model"
+
+M3GNET_DIR_PATH = Path(__file__).parent / "models" / "M3GNet-MP-2021.2.8-DIRECT-PES/"
+M3GNET_MODEL_PATH = (
+    Path(__file__).parent / "models" / "M3GNet-MP-2021.2.8-DIRECT-PES/model.pt"
+)
+M3GNET_POTENTIAL = load_model(path=M3GNET_DIR_PATH)
+
+CHGNET_PATH = Path(__file__).parent / "models" / "chgnet_0.3.0_e29f68s314m37.pth.tar"
```

Comment:
Would it be cleaner to define:
Would it be cleaner to define:
```suggestion

MODEL_PATH = Path(__file__).parent / "models"

MACE_MP_PATH = MODEL_PATH / "mace_mp_small.model"

MACE_OFF_PATH = MODEL_PATH / "MACE-OFF23_small.model"

M3GNET_DIR_PATH = MODEL_PATH / "M3GNet-MP-2021.2.8-DIRECT-PES"
M3GNET_MODEL_PATH = M3GNET_DIR_PATH / "model.pt"
M3GNET_POTENTIAL = load_model(path=M3GNET_DIR_PATH)

CHGNET_PATH = MODEL_PATH / "chgnet_0.3.0_e29f68s314m37.pth.tar"
```